q2_feature_classifier/_custom.py

```diff
@@ -0,0 +1,70 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2017, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import numpy
+from sklearn.base import BaseEstimator, ClassifierMixin, clone  # noqa
+from sklearn.utils.validation import check_X_y, check_array, check_is_fitted  # noqa
+from sklearn.naive_bayes import MultinomialNB
+from sklearn.preprocessing import LabelEncoder
+
+
+class LowMemoryMultinomialNB(MultinomialNB):
+    def __init__(self, alpha=1.0, fit_prior=True, class_prior=None,
+                 chunk_size=-1):
```

Comment:
Thanks @jairideout, I agree that we need to document this behaviour. Where is the best place? In the docstring? In the parameter help string? In the API documentation (does it exist yet?)? All of those places?